.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: java
+jdk: oraclejdk9
+install: ./gradlew --version
+script: ./gradlew --continue --init-script gradle/init-scripts/build-scan.init.gradle.kts --scan --stacktrace sanityCheck
```

Comment:
When we share the gradle user home, I guess we can enable the build cache, right?

Reply:
Do you mean the local cache? I don't think zipping its hundreds of megabytes all up and uploading/downloading it is worth it. It will also make it impossible to clean up if we always download all of it for every build.

Reply:
I don't think downloading everything from bintray is faster...

Let's try it out and see how bad it gets. It would still be cleaned up since we store the access times in files and do not use the modification/access time of the files, right?

Reply:
Yes, I mean the local cache. I think we should try it out...

Reply:
Agreed.